src/main/java/org/sefglobal/scholarx/service/SimpleJavaMail.java

```diff
@@ -0,0 +1,23 @@
+package org.sefglobal.scholarx.service;
+
+import org.simplejavamail.api.email.Email;
+import org.simplejavamail.email.EmailBuilder;
+
+public class SimpleJavaMail {
+
+    public void sendEmail(String name,String emailAddress, String subject, String message, boolean showButton) {
+
+        String username = "pasindur2@gmail.com";
+
+        Email email = EmailBuilder.startingBlank()
+                .from("Sustainable Education Foundation", username)
```

Comment:
I think it automatically gets the sender's username and email from the mail builder instance. If so, we don't have to manually set it here.